api/openapi.gen.yaml

```diff
@@ -930,7 +948,7 @@ info:
     description: Provisioning service API
     license:
         name: GPL-3.0
-    version: 1.7.0
+    version: 1.8.0
 paths:
     /availability_status/sources:
         post:
```

Comment:
As you can see, no changes only new fields (the change hunk is just a doc example).

Reply:
I guess it still needs update of the API client in IQE :)